### PR TITLE
VIT-26: Licensed Technologies.

### PR DESCRIPTION
**Pull request by jsteven97**
# Task [VIT-26](https://manati.atlassian.net/browse/VIT-26)
Creation of the "Tecnologías licenciadas" webform.

# Installation
- git pull in this branch
- Run the following command
`ahoy drush cr && ahoy drush cim -y`

# Test
- Login as admin user.
  - Username: admin.
  - Password: admin.
- Go to the following web path: 
  - ``http://vitaminau.docker/form/webform-licensed-technologies.``
- Once on that page, you should be able to see a form that contains all the fields specified in the Jira:
  - [VIT-26](https://manati.atlassian.net/secure/RapidBoard.jspa?rapidView=104&projectKey=VIT&modal=detail&selectedIssue=VIT-26).
- Fill all the fields of the form and press the save button.
- Once you press it your information should be saved and the form should redirect you to a thank you page.
- To complete a form you have to add a term in the "unit" taxonomy, at this address: ``http://vitaminau.docker/admin/structure/taxonomy/manage/unidad/add`` you can add an element with the name "test".
- Save your draft with the button Save Draft and log in again in another browser or incognito, a "Este formulario ha sido precargado con la información almacenada en el borrador." message should appear.
- if you want check your response or your draft you can go this path ``http://vitaminau.docker/admin/structure/webform/manage/webform_licensed_technologies/results/submissions``  and search your answer.
- To test the required fields, leave a blank, which has a red "*" and the same form will tell you that it has not been completed yet.

# Acceptance criteria
- Only authenticated users can access the form.
- The following fields are not required.
  - Restricciones.
  - ¿Algo que agregar?.
  - PDF relevante.
  - ¿Cómo calificaría este formulario?
  - ¿Qué mejoraría?.
- When the required fields are empty, you should see a message, indicating that it must be filled

### CODE REVIEW

**Comment by jsteven97** [comment]
Ready @dharizza , I confuse "Requerimientos" with "Restricciones" filed.

File: config/sync/language.negotiation.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ url:
   source: path_prefix
   prefixes:
     es: es
+    '': null
```
Comment (dharizza) [inline config/sync/language.negotiation.yml:7]: @jsteven97 can you remove this change? we're not really doing things related so this shouldn't be committed. 
Also, just noticed the form name is not correct, you have it as "Tecnologías licenciadas" but it should be "Tecnologías licenciables", please change.

**Comment by dharizza** [comment]
@jsteven97 we need the field "Unidad" from the "Generadores" fieldset to be a select list that uses the vocabulary "Unidad", instead of a simple textfield. Can you do the change, please?

**Comment by jsteven97** [comment]
Ready @dharizza, I make all the changes and update in relation to the changes in Excel

**Pull request merged**